uqcsbot/scripts/jobs.py

```diff
@@ -0,0 +1,87 @@
+"""
+Monitors #jobs-bulletin and reminds employers and users of their rights and responsibilities.
+"""
+from uqcsbot import bot
+import time
+
+MESSAGE_PAUSE = 2   # Number of seconds between sending bot messages
+FAIR_WORK_INTERNSHIPS_INFO = "https://www.fairwork.gov.au/pay/unpaid-work/work-experience-and-internships"
+CODE_OF_CONDUCT = "https://github.com/UQComputingSociety/code-of-conduct"
+UQCS_EMAIL = "mailto:contact@uqcs.org.au"
+WELCOME_MESSAGES = [    # Welcome messages sent to new members
+    "#jobs-bulletin is a little different to your average UQCS :slack: channel and has a few extra rules:",
+    "*Rules for Everyone* \n1. Only employers advertising jobs are allowed to post in this channel.\n"
+    "2. All discussion about the posted jobs must take place in the #jobs-discussion channel or by direct message "
+    "with the person posting the advertisement. Please be respectful when interacting with companies and sponsors.",
+    "*Additional Rules for Employers Posting Jobs/Internship Opportunities:*\n"
+    f"3. We take the rights of our members and associates seriously. If you are posting an unpaid position, please be"
+    f" up front about the lack of remuneration and *mindful of* <{FAIR_WORK_INTERNSHIPS_INFO}|*your obligations*> "
+    "under the _Fair Work Act (2009)_ :fairwork:. \n_tldr: if an intern (whether called that or not) adds value to "
+    "(or 'does productive work' for) your business, they must be remunerated with either a fair wage or equity_. If you"
+    " ignore these warnings, please expect to face criticism from the community (we will protect our members from being"
+    " exploited).",
```

Comment:
Is it worth mentioning that unpaid positions should be approved by UQ if the applicant is a student within the EAIT school? https://www.eait.uq.edu.au/engineering-professional-practice-unpaid-placements